prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,38 @@
+var number1 = 35;
```

Comment:
you want to update all of these variable declarations to use the `const` keyword.